web/frontend/src/app/component/hierarchy-display/hierarchy-display.component.ts

```diff
@@ -82,11 +91,11 @@ export class HierarchyDisplayComponent implements OnInit {
       tableHeight = bodyHeight;
     }
     this.hierarchyTable.scrollHeight = tableHeight - 200 + 'px';
-  };
+  }
 
   // Handler for selecting a tree node
   treeTableNodeSelected(event) {
-    console.info('treeTableNodeSelected', event);
+    console.log('treeTableNodeSelected', event);
     // Handle selecting for more data for top level
```

Comment:
I'd recommend removing these log statements if we don't need them anymore.  the application overall is pretty quiet and I think keeping it that way is good.

Reply:
This was there before, I just changed it to log instead of info since the latter is not supported in TS. I'll remove this comment